semantic/tasks/admin/components/create.js

```diff
@@ -0,0 +1,331 @@
+/*******************************
+     Create Component Repos
+*******************************/
+
+/*
+ This will create individual component repositories for each SUI component
+
+  * copy component files from release
+  * create commonjs files as index.js for NPM release
+  * create release notes that filter only items related to component
+  * custom package.json file from template
+  * create bower.json from template
+  * create README from template
+  * create meteor.js file
+*/
+
+var
+  gulp            = require('gulp'),
+
+  // node dependencies
+  console         = require('better-console'),
+  del             = require('del'),
+  fs              = require('fs'),
+  path            = require('path'),
+  runSequence     = require('run-sequence'),
+
+  // admin dependencies
+  concatFileNames = require('gulp-concat-filenames'),
+  debug           = require('gulp-debug'),
+  flatten         = require('gulp-flatten'),
+  git             = require('gulp-git'),
+  jsonEditor      = require('gulp-json-editor'),
+  plumber         = require('gulp-plumber'),
+  rename          = require('gulp-rename'),
+  replace         = require('gulp-replace'),
+  tap             = require('gulp-tap'),
+
+  // config
+  config          = require('../../config/user'),
+  release         = require('../../config/admin/release'),
+  project         = require('../../config/project/release'),
+
+  // shorthand
+  version         = project.version,
+  output          = config.paths.output
+
+;
+
+
+module.exports = function(callback) {
+  var
+    stream,
+    index,
+    tasks = []
+  ;
+
+  for(index in release.components) {
+
+    var
+      component = release.components[index]
+    ;
+
+    // streams... designed to save time and make coding fun...
+    (function(component) {
+
+      var
+        outputDirectory      = path.join(release.outputRoot, component),
+        isJavascript         = fs.existsSync(output.compressed + component + '.js'),
+        isCSS                = fs.existsSync(output.compressed + component + '.css'),
+        capitalizedComponent = component.charAt(0).toUpperCase() + component.slice(1),
+        packageName          = release.packageRoot + component,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 10 spaces but found 8.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5182945)